content/en/docs/contributing/pull-requests.md

```diff
@@ -68,14 +68,14 @@ A([fa:fa-user New<br>Contributor]) --- id1[(open-telemetry/opentelemetry.io<br>G
 subgraph tasks[Changes using GitHub]
 direction TB
     0[ ] -.-
-    1[1. Edit this page] --> 2[2. Use GitHub markdown<br>editor to make changes]
-    2 --> 3[3. fill in Propose file change]
+    1[1\. Edit this page] --> 2[2\. Use GitHub markdown<br>editor to make changes]
+    2 --> 3[3\. fill in Propose file change]
 
 end
 subgraph tasks2[ ]
 direction TB
-4[4. select Propose file change] --> 5[5. select Create pull request] --> 6[6. fill in Open a pull request]
-6 --> 7[7. select Create pull request]
+4[4\. select Propose file change] --> 5[5\. select Create pull request] --> 6[6\. fill in Open a pull request]
+6 --> 7[7\. select Create pull request]
```

Comment:
```suggestion
4[4\. Select Propose file change] --> 5[5\. Select Create pull request] --> 6[6\. Fill in Open a pull request]
6 --> 7[7\. Select Create pull request]
```

Reply:
Thank you.
I changed these characters in a bellow commit.

https://github.com/open-telemetry/opentelemetry.io/pull/6253/commits/c532ee535dcccac4ff377b347c55d3fd0c72484e